dhctl/pkg/preflight/check_staticinstances_with_sshcredentials.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2025 Flant JSC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package preflight
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"net"
+	"strings"
+
+	"golang.org/x/crypto/ssh"
+	"gopkg.in/yaml.v3"
+
+	"github.com/deckhouse/deckhouse/dhctl/pkg/util/input"
+)
+
+type SSHCredential struct {
```

Comment:
let to add normal versions struct from CAPS API https://github.com/deckhouse/deckhouse/tree/main/modules/040-node-manager/images/caps-controller-manager/src/api/deckhouse.io to pkg/apis and use this structs